README.md

```diff
@@ -101,10 +101,10 @@ charmhub.
               github_token: ${{ secrets.GITHUB_TOKEN }}
     ```
 
-    a branch name with `upload-charm-docs/migrate` will be created and a pull 
-    request named `[upload-charm-docs] Migrate charm docs` will be created 
-    towards the working branch the workflow was triggered with. 
-    In order to ensure that the branches can be created successfully, please 
+    a branch name with `upload-charm-docs/migrate` will be created and a pull
+    request named `[upload-charm-docs] Migrate charm docs` will be created
+    towards the working branch the workflow was triggered with.
```

Comment:
s/towards the working branch/targeting the branch?
It matches better the git terminology

Reply:
Actually, this is outdated, I'll update it on the next push